CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ For details about compatibility between different releases, see the **Commitment
 - Support for loading end device template from Device Repository when importing devices using a CSV file.
 - Experimental support for normalized payload.
 - Decoded payloads are now visible for downlinks in the Console.
+- Extra validations were added before sending internal notifications.
```

Comment:
This is strictly for us - no need to add it to changelog (end users do not 'feel' it).